srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/EditRiskModelTest.java

```diff
@@ -49,12 +55,31 @@ public final void testEditRiskModel()
         
     	editRiskModel.setModelName( "NewModelName" );
     	
-        assertEquals( editRiskModel.getModelName(), "NewModelName" );
-        assertEquals( editRiskModel.getMaxDisplayNameLength(), DisplayNameConditions.DISPLAY_NAME_MAX );
+        assertEquals( "NewModelName", editRiskModel.getModelName() );
+        assertEquals(  DisplayNameConditions.DISPLAY_NAME_MAX, editRiskModel.getMaxDisplayNameLength() );
         
         editRiskModel.applyChanges();
         
         RiskModel rm = editRiskModel.getRiskModel();
-        assertEquals("NewModelName", rm.getDisplayName() );
+        assertEquals( "NewModelName", rm.getDisplayName() );
+    }
+
+    @Test
+    public final void testGetTermSummaries()
+    {
+    	final EditRiskModel editRiskModel = createEditRiskModel( "Thoracic 30-day Mortality" );
```

Comment:
Everywhere you use this method, the display name is the same. I would suggest getting rid of the parameter and just making the test risk model display name the same so that all tests can use it without having to add the parameter in.